dbms/src/Common/TiFlashMetrics.h

```diff
@@ -107,20 +107,20 @@ namespace DB
     M(tiflash_storage_command_count, "Total number of storage's command, such as delete range / shutdown /startup", Counter,              \
         F(type_delete_range, {"type", "delete_range"}), F(type_ingest, {"type", "ingest"}))                                               \
     M(tiflash_storage_subtask_count, "Total number of storage's sub task", Counter,                                                       \
-        F(type_delta_merge, {"type", "delta_merge"}),                                                                                     \
-        F(type_delta_merge_fg, {"type", "delta_merge_fg"}),                                                                               \
-        F(type_delta_merge_fg_rpc, {"type", "delta_merge_fg_rpc"}),                                                                       \
+        F(type_delta_merge_bg, {"type", "delta_merge_bg"}),                                                                               \
         F(type_delta_merge_bg_gc, {"type", "delta_merge_bg_gc"}),                                                                         \
+        F(type_delta_merge_fg, {"type", "delta_merge_fg"}),                                                                               \
+        F(type_delta_merge_manual, {"type", "delta_merge_manual"}),                                                                       \
```

Comment:
Does grafana need to adjust?

Reply:
In Grafana expressions are using "delta_merge|seg" as matchers, so there is no need to adjust.